frontend/src/routes/Bookshelf.jsx

```diff
@@ -1,23 +1,34 @@
 import '../styles/Bookshelf.css';
-
 import { useState, useEffect } from 'react';
 import Container from 'react-bootstrap/Container';
-import DocumentsCards from '../components/DocumentsCards';
+import FutureDocument from '../components/FutureDocument.jsx';
+import Graph from '../components/Graph.jsx';
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
 
-function Bookshelf({backend, user}) {
-  const [documents, setDocuments] = useState([]);
+function Bookshelf({ createOn, backend, user }) {
```

Comment:
`createOn` is always an empty list. So there is no reason to pass it as a parameter.